scripts/support/README.rst

```diff
@@ -45,6 +45,9 @@ the support account, encrypt the password and print the encrypted password to co
 Installation
 ============
 
+Some scripts of this folder use *osclients.py*. That file can be copied from
+*utils/osclients.py*.
```

Comment:
And why not just import correctly from the directory in which you have the code? Asuming that you clone all the repo, you will have all the code of the glancesync, won't?